tests/beaker_tests/cisco_bgp_af/test_bgpaf.rb

```diff
@@ -315,6 +315,9 @@ def cleanup(agent)
   test_harness_run(tests, :title_patterns_1)
   test_harness_run(tests, :title_patterns_2)
   test_harness_run(tests, :title_patterns_3)
-  test_harness_run(tests, :l2vpn_evpn) unless nexus_i2_image
+  test_harness_run(tests, :l2vpn_evpn) unless nexus_image['I2']
+
+  # -----------------------------------
+  resource_absent_cleanup(agent, 'cisco_bgp')
```

Comment:
Looks like this line stayed in post-merge. Should `resource_absent_cleanup` be removed?